addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -28,12 +28,13 @@
                                 tabindex="-1">
                                 <t t-if="column.hasLabel and column.widget !== 'handle'">
                                 <t t-set="isNumeric" t-value="isNumericColumn(column)"/>
-                                    <div t-att-class="{'d-flex': true, 'flex-row-reverse': shouldReverseHeader(column)}"
+                                    <div class="d-flex"
                                         t-att-data-tooltip-template="isDebugMode ? 'web.FieldTooltip' : 'web.ListHeaderTooltip'"
                                         t-att-data-tooltip-info="makeTooltip(column)">
-                                        <span class="d-block min-w-0 text-truncate flex-grow-1" t-att-class="isNumeric ? 'o_list_number_th' : ''"
+                                        <span class="d-block min-w-0 text-truncate flex-grow-1 flex-shrink-1" t-att-class="isNumeric ? 'o_list_number_th' : ''"
                                               t-esc="column.label"/>
-                                        <i t-att-class="getSortableIconClass(column)"/>
+                                        <span class="d-block o_list_header_label_spacer"></span>
```

Comment:
- `<span>` are non-semantic *inline* elements.
- `<div>` are non-semantic *block* elements.

So, `<span class="d-block">` is a longer way for `<div>` :wink: 
I think you can simplify (I haven't checked if it breaks some selector, though...)